nomad/vault.go

```diff
@@ -47,11 +47,60 @@ const (
 	// vaultRevocationIntv is the interval at which Vault tokens that failed
 	// initial revocation are retried
 	vaultRevocationIntv = 5 * time.Minute
+
+	// vaultCapabilitiesLookupPath is the path to lookup the capabilities of
+	// ones token.
+	vaultCapabilitiesLookupPath = "/sys/capabilities-self"
+
+	// vaultTokenRenewPath is the path used to renew our token
+	vaultTokenRenewPath = "auth/token/renew-self"
+
+	// vaultTokenLookupPath is the path used to lookup a token
+	vaultTokenLookupPath = "auth/token/lookup"
```

Comment:
You should add `auth/token/lookup-self` as a lookup possibility. Most tokens can be looked up using that path as it's in the `default` policy, so if `auth/token/lookup` doesn't work you should use that instead (and probably prefer it).